insights/specs/__init__.py

```diff
@@ -289,6 +289,7 @@ class Specs(SpecSet):
     nfs_exports = RegistryPoint()
     nginx_conf = RegistryPoint(multi_output=True)
     nmcli_dev_show = RegistryPoint()
+    nmcli_conn_show = RegistryPoint()
```

Comment:
Please move this spec ahead of the `nmcli_dev_show` for alphabet order.

Reply:
acknowledged, updated in the code in latest PR